runtime/tutor

```diff
@@ -1516,6 +1516,106 @@ letters! that is not good grammar. you can fix this.
 
 
 
+=================================================================
+=              CHAPTER 14.1 RUN SHELL COMMANDS                  =
+=================================================================
+
+ Shell commands can be run from within helix using the command
+ :run-shell-command , also aliased as :sh . This will execute the
+ command in the parent terminal, and print the output in a new
+ popup.
```

Comment:
```suggestion
 Run shell commands directly within Helix
 using :run-shell-command, also aliased as :sh. This command
 executes in the parent terminal and prints the output in a new
 popup window.
```
Remove passive voice and redundancies, capitalize Helix, clarify slightly. I suggest you go through the document and make this kind of adjustment. See how much text you can get rid of and keep the same meaning.